src/tomato/drivers/biologic/main.py

```diff
@@ -1,17 +1,20 @@
 import logging
 import multiprocessing
 import time
-from filelock import FileLock
 
 from datetime import datetime, timezone
 
+from .kbio import kbio_types as KBIO
 from .kbio_wrapper import (
+    KBIO_api_wrapped,
     get_kbio_techpath,
     payload_to_ecc,
     parse_raw_data,
-    get_kbio_api,
 )
 
+# number of attempts to perform a given operation before giving up
+N_ATTEMPTS = 120
```

Comment:
This could be a driver-level setting, but would have to be passed into the functions similar to `dllpath`.

Reply:
Changed this and the number of required consecutive stop statuses to driver-level settings